components/Cards/SchoolCard/__tests__/FrontSchoolCard.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import FrontSchoolCard from '../FrontSchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
+    zip: '94111',
+  },
+];
+
+describe('FrontSchoolCard', () => {
+  it('should render with required props', () => {
```

Comment:
I'd like to see tests similar to `Button.test.js` line 65 "call props.onClick when button is clicked" but for the callback on both front and back card tests.

Reply:
I outlined an example test in the BackSchoolCard.test.js file 🕺